gen/KubernetesWatchGenerator/ModelExtensions.cs.template

```diff
@@ -3,9 +3,12 @@ namespace k8s.Models
 {{#.}}
     public partial class {{GetClassName . }} : IKubernetesObject
     {
-        public const string KubeApiVersion = "{{GetApiVersion . }}";
-        public const string KubeKind = "{{GetKind . }}";
-        public const string KubeGroup = "{{GetGroup . }}";
+        private const string kubeApiVersion = "{{GetApiVersion . }}";	
+        private const string kubeKind = "{{GetKind . }}";
+        private const string kubeGroup = "{{GetGroup . }}";	
+        public string KubeApiVersion { get { return kubeApiVersion;} }		
```

Comment:
In-order to implement a generic lister, we need a way to find the ApiVersion and Kind from a given Kubernetes object. Currently, we don't have a way to do that (even when using reflection) because consts are treated as static members and hence not available from the object instances. Hence had to convert this to get to gather this from a instance of the Kubernetes object.